tokio-trace-fmt/src/lib.rs

```diff
@@ -135,12 +138,12 @@ where
     fn enter(&self, span: &span::Id) {
         // TODO: add on_enter hook
         let span = self.clone_span(span);
-        span::Context::push(span);
+        span::Context::<N>::push(span);
```

Comment:
I think instead of doing this, we should just move the `push` and `pop` functions to free functions in the `span` module; they really don't need to be on `Context`...